common/templates/common/blocks/video.html

```diff
@@ -0,0 +1,7 @@
+<video
+	class="page-video"
+	{% if self.muted_autoplay_and_loop %}autoplay muted loop{% else %}controls{% endif %}
```

Comment:
I think we also want `playsinline` here as seen here: https://github.com/freedomofpress/freedom.press/pull/767/files

```suggestion
	{% if self.muted_autoplay_and_loop %}autoplay muted loop playsinline{% else %}controls{% endif %}
```